core/src/main/java/org/apache/iceberg/ManifestLists.java

```diff
@@ -21,51 +21,79 @@
 import java.io.IOException;
 import java.util.List;
 import org.apache.iceberg.avro.Avro;
+import org.apache.iceberg.avro.AvroIterable;
 import org.apache.iceberg.exceptions.RuntimeIOException;
 import org.apache.iceberg.io.CloseableIterable;
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.OutputFile;
+import org.apache.iceberg.relocated.com.google.common.annotations.VisibleForTesting;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 
 class ManifestLists {
   private ManifestLists() {}
 
   static List<ManifestFile> read(InputFile manifestList) {
-    try (CloseableIterable<ManifestFile> files =
-        Avro.read(manifestList)
-            .rename("manifest_file", GenericManifestFile.class.getName())
-            .rename("partitions", GenericPartitionFieldSummary.class.getName())
-            .rename("r508", GenericPartitionFieldSummary.class.getName())
-            .classLoader(GenericManifestFile.class.getClassLoader())
-            .project(ManifestFile.schema())
-            .reuseContainers(false)
-            .build()) {
-
+    try (CloseableIterable<ManifestFile> files = manifestFileIterable(manifestList)) {
       return Lists.newLinkedList(files);
-
```

Comment:
Unnecessary whitespace change.

Reply:
I think it was an artifact of how diff chose to show the difference, but I add a blank line.